modules/images/auto-sizes/load.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Module Name: Auto Sizes for Lazy-loaded Images.
+ * Description: Implements auto sizes for lazy loaded images.
+ * Experimental: Yes
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+// Define the constant.
+if ( defined( 'IMAGE_AUTO_SIZES' ) ) {
+	return;
+}
+
+define( 'IMAGE_AUTO_SIZES', 'Performance Lab ' . PERFLAB_VERSION );
```

Comment:
Maybe worth clarifying in the name that it is a version.
```suggestion
if ( defined( 'IMAGE_AUTO_SIZES_VERSION' ) ) {
	return;
}

define( 'IMAGE_AUTO_SIZES_VERSION', 'Performance Lab ' . PERFLAB_VERSION );
```

Reply:
[a5e4afa](https://github.com/WordPress/performance/pull/904/commits/a5e4afa644c085fc9367b8596b9c17cae7f91d7b)